### PR TITLE
Feature/m0078 ddg 306 document mmm 5.2.0 new page

### PR DESCRIPTION
**Pull request by frivieremicroej**
@seonmej @dalgassimou  Please review the content.
We will then merge it in sync with MicroEJ SDK 20.07 Getting Started publication
Thanks

### CODE REVIEW

File: ApplicationDeveloperGuide/mmm.rst [hunk 1]
```diff
@@ -23,6 +23,15 @@ engineers from the difficult task of computing module dependencies.
...
+The following schema introduces the main concepts detailed in this chapter.
+
+.. figure:: images/mmm_flow.png
+   :alt: MMM Overview
+   :align: center
+   :scale: 70%
+
+   MMM Overview
+
 MMM is based on of the following tools:
```
Comment (seonmej) [inline ApplicationDeveloperGuide/mmm.rst:35]: Remove `of`.

File: ApplicationDeveloperGuide/mmm.rst [hunk 2]
```diff
@@ -44,18 +53,18 @@ resolution. Please consult the MMM specification available on
...
+-----------------------
 
 In MicroEJ SDK, a new MicroEJ module project is created as following:
 
 - Select :guilabel:`File` > :guilabel:`New` > :guilabel:`Project...`,
-- Select :guilabel:`EasyAnt` > :guilabel:`EasyAnt Project`,
+- Select :guilabel:`MicroEJ` > :guilabel:`MicroEJ Module Project` [#warning_check_former_sdk_versions]_,
 - Fill the module information (project name, module organization, name and revision),
-- Select one of the suggested skeleton,
+- Select one of the suggested skeleton depending on the desired :ref:`module nature <module_nature_skeleton_mapping>`,
```
Comment (seonmej) [inline ApplicationDeveloperGuide/mmm.rst:64]: skeleton**s**

File: ApplicationDeveloperGuide/mmm.rst [hunk 3]
```diff
@@ -65,14 +74,16 @@ The project is created and default files and directories are generated by the Mi
   - Select :guilabel:`File` > :guilabel:`New` > :guilabel:`Other...`,
   - Select :guilabel:`EasyAnt` > :guilabel:`EasyAnt Skeleton`.
 
+.. [#warning_check_former_sdk_versions] If using MicroEJ SDK versions lower than ``5.2.0``, please refer to the :ref:`following section <mmm_former_sdk>`.
```
Comment (seonmej) [inline ApplicationDeveloperGuide/mmm.rst:77]: The `(1, 2, 3)` generated seems weird. I have to click several times on each link to understand that there are 3 references to this same warning.

File: ApplicationDeveloperGuide/repository.rst [hunk 4]
```diff
@@ -1,3 +1,5 @@
+.. _module_repository:
+
 Module Repository
 =================
```
Comment (seonmej) [inline ApplicationDeveloperGuide/repository.rst:5]: `A module repository takes its input modules from other of repositories […]`. Remove `of`

File: ApplicationDeveloperGuide/repository.rst [hunk 5]
```diff
@@ -20,17 +22,17 @@ It can also be imported in `MicroEJ Forge <https://www.microej.com/product/forge
 Create a Repository Project
 ---------------------------
 
-In MicroEJ SDK, first :ref:`create a new module project <mmm_module_skeleton>` using the ``artifact-repository`` skeleton.
+In MicroEJ SDK, first create a new :ref:`module project <mmm_module_skeleton>` using the ``artifact-repository`` skeleton.
```
Comment (seonmej) [inline ApplicationDeveloperGuide/repository.rst:25]: I think that the `\`` must be before `:ref:`.

Comment (seonmej) [reply ApplicationDeveloperGuide/repository.rst:25]: \` before :ref:

File: ApplicationDeveloperGuide/repository.rst [hunk 6]
```diff
@@ -20,17 +22,17 @@ It can also be imported in `MicroEJ Forge <https://www.microej.com/product/forge
...
 - The ``module.ivy`` file describes how to build repository and lists the module dependencies that will be included in this repository.
 
 Configure Resolver for Input Modules 
 ------------------------------------
 
-MicroEJ Module Manager (MMM) needs to fetch dependencies to build the module repository. 
+MicroEJ Module Manager (MMM) needs to import dependencies to build the module repository. 
 The location fetched by MMM is defined by a resolver.
 The resolver is configured with the parameter ``bar.populate.from.resolver``. The preset value is the resolver
 provided by default in MicroEJ SDK configuration, which is connected to :ref:`MicroEJ Central Repository <central_repository>`.
```
Comment (seonmej) [inline ApplicationDeveloperGuide/repository.rst:38]: `The MicroEJChainResolver is an URL resolver defined defined `
- `a URL`
- remove one `defined`

File: ApplicationDeveloperGuide/repository.rst [hunk 7]
```diff
@@ -129,7 +131,7 @@ The build consists of two steps:
 
 1. Gathers all module dependencies. The whole repository content is created
    under ``target~/mergedArtifactsRepository`` folder.
-2. Checks the repository consistency. For each module, it tries to fetch it from this repository
+2. Checks the repository consistency. For each module, it tries to import it from this repository
    and fails the build if at least one of the dependencies cannot be resolved.
 
 The module repository ``.zip`` file is built in the ``target~/artifacts/`` folder.
```
Comment (seonmej) [inline ApplicationDeveloperGuide/repository.rst:137]: Rephrase `This is file is also published possibly with`. The first `is` should be removed. And the `possibly` seems weird.
Something like: `This file may be published along with a CHANGELOG […]`.

File: ApplicationDeveloperGuide/repository.rst (hunk 5 above)
Comment (frivieremicroej) [reply ApplicationDeveloperGuide/repository.rst:25]: no this is the correct syntax

**Pull request merged**